### PR TITLE
footer added to error page

### LINKED ISSUE #126: Create a simple footer exclusively for Error Component
House of Geeks 404 Page should have an image in the footer, Only a single image that of the moon should be displayed.

### PR DESCRIPTION
**Pull request by tusharjain0022**
### fixes #126 
<Do not delete this template>

Description: <provide a description> I've created a separate footer for 404 error page . 

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] I've tested my code on a Chromium based browser.
- [ ] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial/Screenshot
<provide a relevant screenshot or any testimonial for the solved issue if applicable>

Screenshot (4)

### CODE REVIEW

**Comment by tusharjain0022** [comment]
I tried `#EE2A7B` , `#000018`, `#1EABD6` on the footer but ended up with this  `#420C33` color. Is this color alright? 
@ankiiitraj  @EmperorYP7 @hariom1625 What are your views on this? You can try different colors via inspect tool in the build preview and changing the background-color for the `.footer` class.

File: client/src/components/HOC/withScrollToTop.jsx [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 
 const ScrollToTop = (Component) => {
     const NewComponent = () => {
         useEffect(() => {
             window.scrollTo(0, 0);
         }, []);
         return(
+            <div>
             <Component />
+            <Footer/>
```
Comment (anktrj) [inline client/src/components/HOC/withScrollToTop.jsx:12]: Clever Move 🎉

**Review by ankiiitraj — APPROVED** [review]

**Review by hariom1625 — COMMENTED** [review]
The current color gradient looks cool!! Nice work @tusharjain0022  :-)

Have a look at this too. The screenshot is for iPad and iPad pro. 
Screenshot (201)

**Review by EmperorYP7 — APPROVED** [review]
Looks cool @tusharjain0022. Just a little more adjustments as suggested by @hariom1625 and you're good to go! 🤩

**Comment by tusharjain0022** [comment]
> The current color gradient looks cool!! Nice work @tusharjain0022 :-)
> 
> Have a look at this too. The screenshot is for iPad and iPad pro.
> Screenshot (201)

Issue resolved 🙌

**Comment by ankiiitraj** [comment]
LGTM

**Review by hariom1625 — APPROVED** [review]

**Pull request merged**